clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioParsers.scala

```diff
@@ -59,4 +59,18 @@ trait ClioParsers {
   implicit val pathParser: ArgParser[Path] = parseWith("path", Paths.get(_))
 
   implicit val uuidParser: ArgParser[UUID] = parseWith("uuid", UUID.fromString)
+
+  implicit val stringParser: ArgParser[String] =
+    SimpleArgParser.from[String]("string") { str =>
+      if (str.startsWith("\"") || str.endsWith("\"")) {
```

Comment:
Can't we disallow quotes anywhere in the input? It seems like we'd also want to disallow single quotes.

Reply:
@danxmoran thought it would be leading/trailing only, and the Jira specified double quotes. Have we seen escaped quotes in the middle of strings in the wild?

Reply:
Yeah, my worry with disallowing quotes anywhere is that the "notes" field might contain them somewhere inside (i.e. I can see myself writing `Deleting because KT said "..."` as a note for a delete operation.

Agreed single-quotes would also be good to check for, but definitely only on the edges (otherwise we break contractions, also in the notes field).

Reply:
In the `Someone said "something"` case, would we still want to allow quotes on the trailing edge of the input?

Reply:
Double-checked with KT, and she said it's OK to just disallow double-quotes entirely as long as single-quotes can work.